examples/bank/bank.go

```diff
@@ -35,6 +36,8 @@ import (
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
+var useTransaction = flag.Bool("use_transaction", true, "Turn off to disable transaction.")
```

Comment:
doesn't really matter, but we've generally opted to use dashes (`use-transactions`).